### PR TITLE
Introduction to Android background services

### PR DESCRIPTION
**Pull request by jarlefosen**

### CODE REVIEW

File: _posts/2017-01-13-create-android-background-service.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+ghname: jarlefosen
+jobtitle: Platform Engineer
+tags: [android, sdk]
+---
+
+## Introduction
+
+<div class="message">
+    How do you create a background service in Android that lives its own life?
+    Could you also communicate with it?
```
Comment (gronnbeck) [inline _posts/2017-01-13-create-android-background-service.md]: Maybe "And can you communicate with it"?

File: _posts/2017-01-13-create-android-background-service.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+</div>
+
+## Android background service?
+
+> I want some stuff to happen in the background of my application, even if my application is closed!
+> You see, I really want some function to be invoked every second or so which tells me the current time.
+> And if the application is open, it should be able to display this time.
```
Comment (gronnbeck) [inline _posts/2017-01-13-create-android-background-service.md]: >  And if the application is open, it should be able to display this time.

Something is off with that sentence. Rephrase to something like "And if the application is open the current time should be displayed"

File: _posts/2017-01-13-create-android-background-service.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+This isn't really a useful scenario, but the process of getting this up and running is the same as other more usefull tasks.
```
Comment (gronnbeck) [inline _posts/2017-01-13-create-android-background-service.md]: Typo "usefull" -> "useful"

File: _posts/2017-01-13-create-android-background-service.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+So how do you create a long running background service in Android?
+
+We want a service that completely runs in the background that doesn't interfere with the applications `main thread`. It should provide an interface to communicate with the application when needed.
```
Comment (gronnbeck) [inline _posts/2017-01-13-create-android-background-service.md]: Merge with the previous question

File: _posts/2017-01-13-create-android-background-service.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+layout: post
+title: Creating background services in Android
+ghavatar: 2993240
+ghname: jarlefosen
+jobtitle: Platform Engineer
+tags: [android, sdk]
+---
+
+## Introduction
+
```
Comment (gronnbeck) [inline _posts/2017-01-13-create-android-background-service.md:11]: Remember to introduce Torbjørn's post on welcome song.

**Review by gronnbeck — CHANGES_REQUESTED** [review]
Feedback are high-level. Requires some changes. I'll digg deeper into the content ones these changes are made. OK? Just add me as reviewer once you I've updated the info.

**Comment by jarlefosen** [comment]
@gronnbeck I have rephrased and introduced Torbjørns blogposts regarding beacons and mobile.
We have however decided to wait a bit before releasing this blogpost.

(Reminder to myself, rename filename date before merging in the future)

**Pull request closed without merging**